tools/cli/internal/changelog/outputfilter/squash_test.go

```diff
@@ -0,0 +1,229 @@
+package outputfilter
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewEntriesMapPerIDAndOperationID(t *testing.T) {
+	testCases := []struct {
+		name    string
+		entries []*Entry
+		want    map[string]map[string][]*Entry
+	}{
+		{
+			name:    "Empty entries",
+			entries: []*Entry{},
+			want:    map[string]map[string][]*Entry{},
+		},
+		{
+			name: "Single entry",
+			entries: []*Entry{
+				{ID: "1", OperationID: "op1"},
```

Comment:
[nit] IDs are not numeric, they are oasdiff changes IDs, e.g. response-required-property-added, can we update to avoid misunderstanding?

Reply:
updated